2011-05-02  Peter Breitenlohner  <peb@mppmu.mpg.de>

	* vlna.w (hexnum, settiestr): Declare as static.
	(settiestr): Drop unused local variable 'c'.

diff -ur vlna-1.5.orig/vlna.w vlna-1.5/vlna.w
--- vlna-1.5.orig/vlna.w	2011-05-02 16:00:12.000000000 +0200
+++ vlna-1.5/vlna.w	2011-05-02 16:02:38.000000000 +0200
@@ -172,14 +172,14 @@
 
 @ Vyøe¹íme konverzi kódu zapsaného za parametrem \.{-x} na string |tiestr|.
 @<Pomocné funkce@>=
-unsigned char hexnum(char c) {
+static unsigned char hexnum(char c) {
   if (c >= '0' && c <= '9') return c - '0';
   if (c >= 'A' && c <= 'F') return c - 'A' + 10;
   if (c >= 'a' && c <= 'f') return c - 'a' + 10;
   printusage (), exit (BAD_OPTIONS);  
 } 
-void settiestr(char *s) {
-  int i, j, c;
+static void settiestr(char *s) {
+  int i, j;
   i = strlen(s);
   if ((i > 2*MAXLEN) || i%2 == 1) printusage (), exit (BAD_OPTIONS);
   tiestrlen = i/2;
